dev/source/docs/submitting-patches-back-to-master.rst

```diff
@@ -112,6 +112,15 @@ PRs are more likely to be merged quickly if:
 
 - The PR clearly states what changes in behaviour are expected
 - Good testing evidence is provided.  This could be graphs of logs recorded before and after the change
-- The PR is discussed on the :ref:`weekly dev call <ardupilot-discord-server>`.  To get the PR discussed add the "DevCallTopic" label.  If you are unable to add the label ping one of the developers on `ArduPilot Discord <https://ardupilot.org/discord>`__
+- Code follows the :ref:`style guide <style-guide>`
+- Each commit in the PR affects only one subsystem and the commit title is prefixed with the subsystem name (e.g. "AP_GPS: correct uBlox logging parameter descriptions")
```

Comment:
point to Tools/gittools/git-subsytems-split.py ?

Reply:
ok, thanks. I've added a link to the script